### DISCUSSION TITLE
Backpack 6 list.blade issue

### DISCUSSION THREAD

**Discussion by Sanjayadol** [Laravel-Backpack/community-forum#603] [Q&A (Help)] [answered]
i encountered a issue when upgrade the Backpack from 5.6 to 6. I GET all the datas in Backpack 5.6 after update receive 10 records it got paginated automatically. I attached the screenshots of both.

1. Backpack 5.6 ( list.blade -> setListView output )

![Screenshot from 2023-08-01 15-09-31](https://github.com/Laravel-Backpack/community-forum/assets/118426203/6f93947f-4b1a-4459-a45f-7b34c5042c73)

2. Update Backpack 6 ( list.blade -> setListView output )

![Screenshot from 2023-08-01 15-04-13](https://github.com/Laravel-Backpack/community-forum/assets/118426203/e8d4cf34-6278-4e92-90bc-a98331412ad8)

**Comment by christmex** [comment]
Hi @Sanjayadol 

Sorry man, but can you please provide us with more information? like what you did inside your setupListOperation()? Also is there a difference inside the setupListOperation when you use v5 and v6? maybe you add something?


Cheers.

**Comment by Sanjayadol** [comment, marked as answer]
Thanks for your reply, I didn't change anything my code. But when I update v6 below got updated by default in setupListOperation. When I removed this line entry counts works properly.
![setuplistoperation](https://github.com/Laravel-Backpack/community-forum/assets/118426203/3b99fa88-c0a2-4ad0-b54c-de1480653683)

**Comment by christmex** [comment]
Hi @Sanjayadol 

That's a feature from backpack in order to make the ListOperation super-fast on big database tables, you can read it here
https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#large-tables-millions-of-entries

That's actually fine if you remove it, but it will be good if you keep that line, to make your performance fast.

**Comment by karandatwani92** [comment]
Hey @Sanjayadol,  It seems like `$this->crud->addClause` is used on the list operation.

You can find more APIs here which may help you solve the issue:
https://backpackforlaravel.com/docs/6.x/crud-api#custom-advanced-queries

if the issue still persists,  let us know with more details and your controller's code.
